specification/network/resource-manager/Microsoft.Network/preview/2021-05-01-preview/networkManagerGroup.json

```diff
@@ -345,15 +505,31 @@
       },
       "description": "Properties of network group"
     },
-    "groupMembersItem": {
+    "StaticMember": {
```

Comment:
Not quite this simple. Static member also needs to inherate from the base class to get things like name, id, type, etc